packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -220,6 +220,20 @@ QUnit.module('options changing', {
         }
     });
 
+    QUnit.test('The validation error should be null after removing the mask (T1214604)', function(assert) {
+        const textBox = $('#textbox').dxTextBox({
+            value: '123',
+            mask: '000000/0009',
+            maskInvalidMessage: 'mask error',
+        }).dxTextBox('instance');
+
+        assert.strictEqual(textBox.option('validationError').message, 'Value is invalid', 'validation message was added');
+
+        textBox.option({ mask: '' });
+
+        assert.strictEqual(textBox.option('validationError'), null);
+    });
+
```

Comment:
I also ask you to add a test with external validation using `validator` to check that external validation error will be kept after `mask` option runtime change.

Reply:
[c0783df5eff72f2e9d00601f6521ea5721a42bf0]